### ISSUE TITLE
I keep receiving Network Error whenever I try to use the client

### ISSUE THREAD

**Issue by exoarmyl** [ORelio/Minecraft-Console-Client#1042]
Every time I try to use the client, I keep receiving a Network Error. I am not using a VPN, am using correct credentials, have a stable internet connection, and am able to play Minecraft with the regular launcher. 

This is what the bug report says:

> Console Client for MC 1.4.6 to 1.15.2 - v1.15.2 - By ORelio & Contributors
> AppVeyor build 278, built on Sun 04/19/2020 from commit 26a7db6
> [MCC] [Settings] Loading Settings from C:\Users\exoar\Downloads\Console Client 2020\Console Client\MinecraftClient.ini
> Resolving mineheros.net...
> Connecting to Minecraft.net...
> Performing request to authserver.mojang.com
> > POST /authenticate HTTP/1.1
> > Host: authserver.mojang.com
> > User-Agent: MCC/1.15.2
> > Content-Type: application/json
> > Content-Length: 172
> > Connection: close
> >
> > {"agent": { "name": "Minecraft", "version": 1 }, "username": [redacted] "password": [redacted], "clientToken": [redacted] }
> Got error code from server: 520
> Minecraft Login failed : Network error.
> [MCC] [AutoRelog] Launching with 3 reconnection attempts
> [MCC] [AutoRelog] Loading messages from file: C:\Users\exoar\Downloads\Console Client 2020\Console Client\kickmessages.txt
> [MCC] [AutoRelog]   Loaded message: Connection has been lost
> [MCC] [AutoRelog]   Loaded message: Server is restarting
> [MCC] [AutoRelog]   Loaded message: Server is full
> [MCC] [AutoRelog]   Loaded message: Too many people
> [MCC] [AutoRelog]   Loaded message: Afk
> [MCC] [AutoRelog] Got disconnected with message: Minecraft Login failed : Network error.
> [MCC] [AutoRelog] Message not containing any defined keywords. Ignoring.
> Not connected to any server. Use '/help' for help.
> Or press Enter to exit Minecraft Console Client.
>

**Comment by ORelio** [comment]
Seems like the request times out on your side.
Cannot reproduce here, I get 200 OK as expected.. 😕

**Comment by exoarmyl** [comment]
So if I were to use an AltLogin bat file and use this script, would it allow for enough time for the client to connect before it times out?
>start /min MinecraftClient.exe [username] [password] [server ip]
timeout 120

**Comment by ORelio** [comment]
A timeout is abnormal because MCC waits 30 seconds for the HTTPS request to complete. If the server does not respond in time, there is likely some blocking somewhere between MCC and the login servers, or on the login server themselves 🤔

**Comment by exoarmyl** [comment]
So, I noticed that the bug report said:
>[MCC] [AutoRelog] Got disconnected with message: Minecraft Login failed : Network error.
>[MCC] [AutoRelog] Message not containing any defined keywords. Ignoring.
So, I tried putting "Network error" in the kickmessage.txt to see what would happen. Now, it just keeps trying to reconnect without any success. However, there's some new information in the bug report which I will paste here to see if you understand any of it lol.

Restarting Minecraft Console Client...
Connecting to Minecraft.net...
Performing request to authserver.mojang.com
> POST /authenticate HTTP/1.1
> Host: authserver.mojang.com
> User-Agent: MCC/1.15.2
> Content-Type: application/json
> Content-Length: 179
> Connection: close
>
> {"agent": { "name": "Minecraft", "version": 1 }, "username": [readcted], "password": [redacted], "clientToken": [redacted] }
System.IO.IOException: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host
   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at MinecraftClient.Protocol.ProtocolHandler.DoHTTPSRequest(List`1 headers, String host, String& result)
   at MinecraftClient.Protocol.ProtocolHandler.DoHTTPSPost(String host, String endpoint, String request, String& result)
   at MinecraftClient.Protocol.ProtocolHandler.GetLogin(String user, String pass, SessionToken& session)

**Comment by ORelio** [comment]
`An existing connection was forcibly closed by the remote host.`: After receiving the login request, the authentication server just closes the connection without responding. There is definitely some filtering there that blocks request emanating from MCC, but only from your location, not mine 🤔

**Comment by exoarmyl** [comment]
Do you have any idea what I may be able to do to try and fix it? I download MCC on my boyfriends laptop and it worked on my internet, as well as his. So, I don't think it's a problem with my internet. I think it's a problem with my laptop.

**Comment by ORelio** [comment]
Just in case, try checking Proxy settings and Antivirus settings. Proxy settings might affect MCC, and Antivirus might have something like "intercept HTTPS" "intercept web traffing" setting that is tampering with MCC in some way, you may need to exempt MCC from this, or try disabling the setting.

**Comment by exoarmyl** [comment]
So, I finally got the console client working by chance. What had happened was my Minecraft launcher recently stopped working, kept saying no connection, and wouldn't let me on any servers. So after doing some Googling, I found a thread where someone said that in their C:\Windows\System32\drivers\etc folder, they had a file named "Host." After deleting that file, my Minecraft launcher immediately started working again, and I was able to go on servers. So, I was like what if this somehow also fixed my MCC and decided to load it up to check if by chance it worked. And, it worked! So apparently, that Host file was somehow blocking my connection to the Minecraft servers or whatever, and by deleting it, I was able to start using the client.

Thanks for your help!
